### PR TITLE
add a created-by component

### PR DESCRIPTION
**Pull request by tylerlee**
Creates a created-by component for easily repeating this section. Not used in every tool but in a few so far, hopefully more.

### CODE REVIEW

**Comment by tylerlee** [comment]
@caseywebdev want to take a look @ this small component?

**Comment by caseywebdev** [comment]
Yup looks like a component

File: bower.json [hunk 1]
```diff
@@ -37,6 +37,6 @@
     "tinycolor": "1.1.2",
     "underscore": "1.7.0",
     "underscore.string": "3.0.2",
-    "velcro": "fbf1e0f3fb520f016da88907896c45030b210eef"
+    "velcro": "3757ea7dbedea6048d1d892b961b365070c79b8a"
```
Comment (caseywebdev) [inline bower.json]: I think I started versioning velcro so we don't have to do this

File: scripts/components/shared/created-by.es6 [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+import React from 'react';
+import {getPictureUrl} from 'entities/account';
+
+export default React.createClass({
+  render: function () {
```
Comment (caseywebdev) [inline scripts/components/shared/created-by.es6]: Add some `propTypes`

File: styles/ui/media.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@import '../config';
+
+.os-media{
+  @include os-media('os-pull-left', 'os-media-body');
```
Comment (caseywebdev) [inline styles/ui/media.scss]: `osw-` not `os-`

File: scripts/components/shared/created-by.es6 [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    return (
+      <div className='os-media'>
```
Comment (caseywebdev) [inline scripts/components/shared/created-by.es6]: `osw-` not `os-`

File: styles/ui/media.scss [hunk 5]
```diff
@@ -0,0 +1,9 @@
+@import '../config';
+
+.os-media{
```
Comment (caseywebdev) [inline styles/ui/media.scss]: space before `{`

File: styles/ui/media.scss (hunk 3 above)
Comment (caseywebdev) [inline styles/ui/media.scss]: actually this note should go on the line above

Comment (tylerlee) [inline styles/ui/media.scss]: applied to both.

File: scripts/components/polls/show.es6 [hunk 6]
```diff
@@ -59,14 +60,15 @@ export default React.createClass({
...
         <h3>{poll.name}</h3>
 
         {this.renderStatus(poll)}
 
-        <p>Created by {poll.creator.display_name}</p>
+        <CreatedBy account={poll.creator} createdAt={poll.created_at} />
 
         {this.renderVoted(poll)}
-
+
```
Comment (caseywebdev) [inline scripts/components/polls/show.es6]: eslint should have caught this

**Comment by tylerlee** [comment]
@caseywebdev changes made.

**Comment by caseywebdev** [comment]
Beautiful :+1:

**Pull request merged**